src/org/traccar/model/Server.java

```diff
@@ -166,4 +168,17 @@ public void setCoordinateFormat(String coordinateFormat) {
         this.coordinateFormat = coordinateFormat;
     }
 
+    private String timezone;
+
+    public void setTimezone(String timezone) {
+        if (timezone != null) {
+            this.timezone = TimeZone.getTimeZone(timezone).getID();
```

Comment:
Added validation here. 
`TimeZone.getTimeZone()` return "GMT" if not recognizes timezone and we get always valid timezone ID in DB